### DISCUSSION TITLE
Can't find Package Development section in new documentation

### DISCUSSION THREAD

**Discussion by andreapozza** [livewire/livewire#8582] [4. General]
I was trying to understand how a package registers its livewire components and found the explanation in the 2.x documentation. https://laravel-livewire.com/docs/2.x/package-dev
I can't find the same page in version 3.x. Is the method always the same?

**Comment by PhiloNL** [comment]
Hi @andreapozza,
In v3 this works the same.

**Comment by andreapozza** [comment]
I'm curious to know why it is not included in the v3.x documentation. Is it being softly deprecated?

**Reply by joshhanley** [reply]
@andreapozza no, probably just an oversight. PRs welcome!

**Reply by andreapozza** [reply]
Not sure these changes are suficient to update the documentation.
https://github.com/livewire/livewire/pull/8632
Let me know if any further changes are needed.
